qir/qat/BUILD

```diff
@@ -423,3 +439,23 @@ ms_cc_binary(
         "//qir/qat:Version",
     ],
 )
+
+pkg_tar(
+    name = "qat-tar",
+    srcs = [":qat"],
+    mode = "0755",
+    package_dir = "/opt",
+    visibility = ["//visibility:public"],
+)
+
+# To enable building Docker images, add this
```

Comment:
Just to clarify for my own understanding, this means a developer would need to modify this file to uncomment these lines before they could use the image build option, correct? Does leaving this on by default cause problems if the environment doesn't have docker, for example?

Reply:
Yes, that is correct. These lines need uncommenting. The reason is that docker builds does not work on native Windows PowerShell and would hence break CI. I have not figured out a way enable this target without it affecting the Windows builds, but it is something I am looking into (low priority though).

Reply:
Makes sense, thanks for the background!